<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFavoriteDialog</class>
 <widget class="QDialog" name="AddFavoriteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加收藏</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="NameLabel">
     <property name="text">
      <string>名字：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="DirComboBox"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="NameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="FileName">
     <property name="text">
      <string>文件夹：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="Btn_newDir">
     <property name="text">
      <string>新建文件夹</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
